cmd/apps/sealed_secret_app.go

```diff
@@ -36,7 +38,6 @@ func MakeInstallSealedSecrets() *cobra.Command {
 		if err := config.SetKubeconfig(kubeConfigPath); err != nil {
```

Comment:
we dont need to set it here, its gets set in the chart app

Reply:
Thanks for mention. I have made the changes. @Waterdrips